app/auth/views/users.py

```diff
@@ -250,28 +250,32 @@ def submit_create_account():
 def send_reset_password_email():
     json_payload = get_json_from_request()
     email_address = json_payload.get('email_address', None)
+    framework = json_payload.get('framework', 'digital-marketplace')
     if email_address is None:
         return jsonify(message='One or more required args were missing from the request'), 400
 
     user = User.query.filter(
         User.email_address == email_address).first()
 
+    app_root_url = current_app.config['{}_APP_ROOT'.format("ORAMS" if framework == "ORAMS" else "MARKETPLACE")]
```

Comment:
I assume these strings will be needed all over the place - not just reset password
you could map these somewhere `{'dm': 'MARKETPLACE', 'orams': 'ORAMS'}`

matching framework-slug to description - if you get the framework-slug passed to you makes it simple